### PR TITLE
init

### PR DESCRIPTION
**Pull request by pom11**
process new exrproxy free_evm_passthrough plugin

```
    - name: HYDRA
      free: True
      chains:
        - name: AVAX
        - name: ETH
```

configure it with `free` key in input yaml

### CODE REVIEW

File: autobuild/examples/xquery-gethEXT-avaxEXT.yaml [hunk 1]
```diff
@@ -46,6 +46,7 @@
     - name: AVAX 
       host: 192.168.32.11 #examaple IP for external avax
     - name: HYDRA
+      free: True
```
Comment (ConanMishler) [inline autobuild/examples/xquery-gethEXT-avaxEXT.yaml:49]: Shouldn't this line default to `False`?
```
      free: True
```

File: autobuild/examples/xquery-gethINT-avaxINT.yaml [hunk 2]
```diff
@@ -48,6 +48,7 @@
       image: avaplatform/avalanchego:v1.7.2
       data_mount_dir: /snode
     - name: HYDRA
+      free: True
```
Comment (ConanMishler) [inline autobuild/examples/xquery-gethINT-avaxINT.yaml:51]: Shouldn't this line default to `False`?
```
      free: True
```

**Review by ConanMishler — CHANGES_REQUESTED** [review]
See embedded comments.

File: autobuild/examples/xquery-gethEXT-avaxEXT.yaml (hunk 1 above)
Comment (pom11) [reply autobuild/examples/xquery-gethEXT-avaxEXT.yaml]: Changed it to `False`

**Review by ConanMishler — APPROVED** [review]

**Review by ConanMishler — APPROVED** [review]
Seems harmless, now that free defaults to `False`.

**Pull request merged**